src/components/Ticket/Ticket.tsx

```diff
@@ -81,6 +48,23 @@ export default function Ticket({
   );
 }
 
+export function DraggableTicket({ id, index, ...ticketProps }: DraggableTicketProps): JSX.Element {
```

Comment:
this `DraggableTicket` is a superset of `Ticket` component. Consumer can individually import either ticket or Draggable ticket, so that you won't need to have flag logic to determine which ticket to show